app/models/folio/item.rb

```diff
@@ -87,6 +87,12 @@ def home_location
       permanent_location.code
     end
 
+    def call_number
+      return '(no call number)' if callnumber.blank?
+
+      callnumber
+    end
```

Comment:
Is it weird to have both`#call_number` and `#callnumber` with two different behaviors? Is there a better name for this value (`item_label`, maybe?) or a better place for this logic (maybe a helper, if it is display-only?)

Reply:
moved to helper